doc/en/interactive.rst

```diff
@@ -33,37 +33,51 @@ interrupt processes started as part of the test environment.
 
 .. note::
 
-    The interactive mode is currently in a Beta stage and under active
-    development. Therefore, features may change in future releases and
-    there will likely be some bugs or not-implemented features right now,
-    for example the filter box does not yet work. Please bear with us while
-    we work through these, and feel free to raise an issue for any bugs
-    you find.
+    The interactive mode is currently in a beta stage and under active development.
+    There are still a few features yet to be implemented, for example the filter
+    box, tree view navigation and timing view, etc. Please bear with us while we
+    work through these, and feel free to raise an issue for any bugs you may find.
 
 How code reloading works
 ------------------------
-By default interactive mode only reloads modules under the directory of the
-main testplan script being run. To be able to reload from other directory, use
-the ``extra_deps`` parameter of ``@test_plan`` decorator. The reload happens on
-reload API and only applies to modules that have either been modified or have
-had its dependencies reloaded.
-
-Please note the ``__main__`` module (i.e. test_plan.py) cannot be reloaded.
-Because of this, it is recommended that test suites are defined in seperate modules.
-Another known limitation is that we are not able to reload module that is under
-namespace package, and the workaround is to add dummy __init__.py file.
+Code-reloading is triggered by reload API request, and there is a reload button
+on interactive mode UI. By default, testplan will look under the directory of
+the main script for modules to reload - those who have either been modified or
```

Comment:
This is not precise, we will look under the directory of the main script and the directories where the tasks are scheduled from.